pkg/plugins/platform/local.go

```diff
@@ -281,9 +279,34 @@ func (l *LocalApp) DockerComposeYAMLPath() string {
 	return l.AppConfig.DockerComposeYAMLPath()
 }
 
+// ComposeFiles returns a list of compose files for a project.
+func (l *LocalApp) ComposeFiles() []string {
+	files, err := util.FindFiles(path.Join(l.AppRoot(), ".ddev"), "docker-compose")
```

Comment:
I would think that the location of .ddev would be exported somewhere. Might be a good time to do it. So much good refactoring here already, and there's no reason for this to be computed every time it's found. Maybe just add AppDdevDir() to l.